### ISSUE TITLE
When web-map-links are rendered on map half opacity fill of bbox feature partially covers data

### ISSUE THREAD

**Issue by santilland** [radiantearth/stac-browser#345]
Not a critical issue but would be great if the fill could somehow be disabled.
One approach could be to check if a web-map-links is being rendered and if yes set to opacity to 0 in the bbox feature fill?

**Comment by m-mohr** [comment]
Will be fixed in the OpenLayers variant of stac-layer, so it is probably not worth implmenting it for Leaflet as we'll migrate anyway.

**Comment by m-mohr** [comment]
Yeah, let's keep it open to ensure once OL migration is done that it really works.

**Comment by m-mohr** [comment]
This is fixed in #531.
